spec/features/workbooks_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.feature "Workbooks management" do
+
+  scenario "Admin creates and deletes a workbook", :js do
+
+    subject = create(:subject)
+
+    visit new_workbook_path
+    expect(page).to have_css 'div.form-group'
+    expect(page).not_to have_link 'Delete'
+    fill_in 'workbook[name]', with: " "
+    click_button 'Save'
+
+    expect(page).to have_css 'div#error_explanation'
+
+    visit new_workbook_path
+    fill_in 'workbook[name]', with: "Workbook1"
+    select subject.name, :from => "workbook[subject_id]"
+    page.execute_script <<-JS
+    $("#workbook_description").val('About workbook1');
+    JS
```

Comment:
You should extract this in a shared helper. Then you'd do like this:

    set_text_in_wysiwyg_editor('#workbook_description', text)